src/backend/oracle-data-api/src/main/resources/occm-openapi-spec.yaml

```diff
@@ -489,6 +489,29 @@ components:
           type: string
         addressProvSeqNo:
           type: string
+        appearanceDtm:
+          type: string
+          format: date
+          x-field-extra-annotation: "@com.fasterxml.jackson.annotation.JsonFormat(pattern=\"yyyy-MM-dd'T'HH:mm:ss'Z'\")"
```

Comment:
I think the format for this should be `format: date-time`

Reply:
Right you are. Great catch. Fixed.